src/main/java/org/prebid/server/proto/openrtb/ext/ExtReqPrebid.java

```diff
@@ -0,0 +1,15 @@
+package org.prebid.server.proto.openrtb.ext;
```

Comment:
This file ins't used anywhere

Reply:
It will be used by bidders to implement parsing from ext.prebid.  
I thing, that this PR is better place for containing that logic. (bc we add new feature and tools for it)